MainUI.cs

```diff
@@ -348,5 +379,41 @@ private void CleanTab()
         }
 
         #endregion
+
+        #region PopularTransactions
```

Comment:
quite often a `#region` indicates that the class is becoming too big to manage. When moving to a different UI, consider splitting this view into several independent components.